network/wsNetwork.go

```diff
@@ -226,12 +223,8 @@ type GossipNode interface {
 	// SubstituteGenesisID substitutes the "{genesisID}" with their network-specific genesisID.
 	SubstituteGenesisID(rawURL string) string
 
-	// GetPeerData returns a value stored by SetPeerData
-	GetPeerData(peer Peer, key string) interface{}
```

Comment:
This and things like DisconnectPeers are implementable with similar semantics on the p2p side as well if we want to but I do agree that it doesn't have to be a part of the interface